packages/repository-tests/src/crud/relations/helpers.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/repository-tests
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {juggler} from '@loopback/repository';
+import {CrudRepositoryCtor} from '../..';
+import {
+  createOrderRepo,
+  createAddressRepo,
+  createCustomerRepo,
+  createShipmentRepo,
+} from './fixtures/repositories';
+
+export function givenBoundCrudRepositories(
+  db: juggler.DataSource,
+  repositoryClass: CrudRepositoryCtor,
+) {
+  // get the repository class and create a new instance of it
+  const customerRepoClass = createCustomerRepo(repositoryClass);
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const customerRepo: any = new customerRepoClass(
```

Comment:
As I commented earlier, I'd like us to find a way how to avoid `any` and use an interface describing the properties & methods of each model-specific repositories.